### PR TITLE
Comply with Google Sign in and Google Drive branding guidelines

### PR DESCRIPTION
**Pull request by lasryaric**
Comply with the Google branding guideline.
They simply said that we were not complying with their branding guidelines. So I read their doc and tried to come up with a list of changes. I hope it's enough, we'll see what they say.

Changes:
- The Google Sign in button must have a white or a blue background.
- The Google Drive logo must be the one they provide. Ours did not have the little red triangle.
- The Google Sign In button must use the Roboto Medium font.

### CODE REVIEW

**Comment by spolu** [comment]
The 32x32 looks a bit smaller did you check it's not too weird in the data source list of Chat or document list of data sources?

File: front/tailwind.config.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ module.exports = {
     "./components/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
+    fontFamily: {
+      roboto: ["Roboto", "sans-serif"],
```
Comment (spolu) [inline front/tailwind.config.js:9]: This is just an additive className right?

**Comment by lasryaric** [comment]
> The 32x32 looks a bit smaller did you check it's not too weird in the data source list of Chat or document list of data sources?

Looks good to me. WDYT?
Screen Shot 2023-06-26 at 17 26 52

**Comment by spolu** [comment]
Agreed!

**Comment by lasryaric** [comment]
> The 32x32 looks a bit smaller did you check it's not too weird in the data source list of Chat or document list of data sources?

Adding this screenshot to help you double check.
Screen Shot 2023-06-26 at 17 28 31

**Comment by lasryaric** [comment]
@spolu I think it's ready for merging. LMK.

**Review by spolu — APPROVED** [review]
LGTM

**Pull request merged**